src/Properties/BaseProperties.php

```diff
@@ -4,6 +4,9 @@
 
 namespace Inpsyde\Modularity\Properties;
 
+/**
+ * phpcs:disable PHPCompatibility.FunctionDeclarations.RemovedImplicitlyNullableParam.Deprecated
```

Comment:
We should not ignore this, but fix the issue. This is about the `string` parameter in the constructor that has a `null` default value.

```diff
    protected function __construct(
        string $baseName,
        string $basePath,
-        string $baseUrl = null,
+        ?string $baseUrl = null,
        array $properties = []
    ) {
```